src/main/kotlin/org/jitsi/jibri/config/JibriConfig.kt

```diff
@@ -155,7 +155,13 @@ data class JibriConfig(
     @JsonProperty("finalize_recording_script_path")
     val finalizeRecordingScriptPath: String? = null,
     @JsonProperty("xmpp_environments")
-    val xmppEnvironments: List<XmppEnvironmentConfig>? = null
+    val xmppEnvironments: List<XmppEnvironmentConfig>? = null,
```

Comment:
This config should be considered deprecated, so let's not add new properties there, that will just make it harder to phase out.